apps/weight-converter/src/App.jsx

```diff
@@ -106,9 +98,16 @@ const parseConversionString = (query) => {
 // This where you can process the query and try to convert it into some meaningful data.
 async function queryToData({ query }) {
   try {
-    let { amount, from, to } = parseConversionString(query);
-    from = mapUnits(from);
-    to = mapUnits(to);
+    let { amount, from, to } = parseConversionString(
+      replaceSynonyms(query, {
+        kilogram: ["kilograms", "kgs", "kg"],
+        milligram: ["milligrams", "mg", "mgs"],
+        tonne: ["tonnes", "tn", "ton"],
+        pound: ["pounds", "lbs", "lb"],
+        ounce: ["ounces", "oz"],
+        stone: ["stones", "st"],
+      })
+    );
```

Comment:
Maybe we should include the keynames in the value array too, because this instant app doesn't work when it comes to query like "5 tonne to kilogram", i.e. when keynames are written.